public/v4/apps/affine.yml

```diff
@@ -0,0 +1,136 @@
+captainVersion: 4
+services:
+  #affine:
+  $$cap_appname:
+    image: ghcr.io/toeverything/affine-graphql:$$cap_affine_version
+    command:
+      ['sh', '-c', 'node ./scripts/self-host-predeploy && node ./dist/index.js']
+    ports:
+      - '3010:3010'
+      - '5555:5555'
+    depends_on:
+      $$cap_appname-redis:
+        condition: service_healthy
+      $$cap_appname-postgres:
+        condition: service_healthy
+    volumes:
+      # custom configurations
+      - $$cap_appname-db-data:/root/.affine/config
+      # blob storage
+      - $$cap_appname-db-data:/root/.affine/storage
+    logging:
+      driver: 'json-file'
+      options:
+        max-size: '1000m'
+    restart: unless-stopped
+    environment:
+      NODE_OPTIONS: "--import=./scripts/register.js"
+      AFFINE_CONFIG_PATH: /root/.affine/config
+      REDIS_SERVER_HOST: srv-captain--$$cap_appname_data-redis
+      DATABASE_URL:  postgres://affine:$$cap_db_affine_password@srv-captain--$$cap_appname-postgres:5432/affine
+      NODE_ENV: production
+      AFFINE_ADMIN_EMAIL: $$cap_affine_admin_email
+      AFFINE_ADMIN_PASSWORD: $$cap_affine_admin_password
+      AFFINE_SERVER_HOST: $$cap_affine_server_host
+      AFFINE_SERVER_PORT: $$cap_affine_server_port
+      AFFINE_SERVER_HTTPS: $$cap_affine_server_https
+      AFFINE_SERVER_EXTERNAL_URL: $$cap_affine_server_external_url
+      MAILER_HOST: $$cap_mailer_host
+      MAILER_PORT: $$cap_mailer_port
+      MAILER_USER: $$cap_mailer_user
+      MAILER_PASSWORD: $$cap_mailer_password
+      MAILER_SENDER: $$cap_mailer_sender
+      # Telemetry allows us to collect data on how you use the affine. This data will helps us improve the app and provide better features.
+      # Uncomment next line if you wish to quit telemetry.
+      # - TELEMETRY_ENABLE=false
+  $$cap_appname-redis:
+    image: redis
+    restart: unless-stopped
+    volumes:
+      - $$cap_appname-db-data:/data
+    healthcheck:
+      test: ['CMD', 'redis-cli', '--raw', 'incr', 'ping']
+      interval: 10s
+      timeout: 5s
+      retries: 5
+  $$cap_appname-postgres:
+    image: postgres
+    restart: unless-stopped
+    volumes:
+      - $$cap_appname-db-data:/var/lib/postgresql/data
+    healthcheck:
+      test: ['CMD-SHELL', 'pg_isready -U affine']
+      interval: 10s
+      timeout: 5s
+      retries: 5
+    environment:
+      POSTGRES_DB: affine
+      POSTGRES_USER: affine
+      POSTGRES_PASSWORD: $$cap_db_affine_password
+      PGDATA: /var/lib/postgresql/data/pgdata
+caproverOneClickApp:
+  variables:
+    - defaultValue: stable
+      description: Check out their Docker page for the valid tags https://github.com/toeverything/AFFiNE/pkgs/container/affine-graphql
+      id: $$cap_affine_version
+      label: Affine version
+      validRegex: /^([^\s^\/])+$/
+
+    - description: Default access domain of AFFiNE. Make sure this matches the URL in browser.	localhost
```

Comment:
I am guess `localhost` was supposed to be the default value and it is just a typo that it's placed at the end of the description?

Reply:
It was the default value. I changed to `(eg. yourdomain.com)`